apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -693,43 +709,28 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
 
   useEffect(() => {
     if (address && isAuthenticated) {
-      getUserByAddress();
+      getUserByAddress().then();
```

Comment:
Why add then here if there is no logic? A non awaited promise should do the same as one with then right?

Reply:
Yes, it should. Stupid mistake! I got confused between `async`/`await` and `then()` or `catch()`, I thought they have the same behavior, forgetting that `then()` or `catch()` takes a function.

In `async/await`, for non-awaited promises it wouldn't handle the rejected case (in case of error) or propagate the error, for better debugging.

[Playground Example](https://www.typescriptlang.org/play?#code/MYewdgzgLgBATgUwFYOFBATACnEBbASwgRgF4YBDCATzGBgAoBKMgPhgG8YAoGGKABa4A7jDAJRAUTi44DAOQAxCgQA2mGDnxEE8pgG5uAX27dQkWADMArqtWW167LkLEylGnUYtS7Lr3gEKGs4MBgAFgAmQxMzcGgYPBUw8ipaemY2Th4+cwTVAksSVOEVK1t7R0wtVwRmQ1z4kHUAOlUQAHMGDgKiowNTKDhqbICAejHKUoJYRBQ0apcdevHJudR0Z21iFb4JwPnNmuWmFsEEMBWjGGAKKGABRgQWf0bIZoQ2zoZnmH2hggdDoIRAYAD8xgCeQ+Xy6ACIAFxIhHrBYYZEIuFMSGmJIES4DIA)